velox/functions/prestosql/StringFunctions.h

```diff
@@ -154,6 +154,36 @@ struct HmacSha512Function {
   }
 };
 
+/// spooky_hash_v2_32(varbinary) -> varbinary
+template <typename T>
+struct SpookyHashV2_32Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TTo, typename TFrom>
+  FOLLY_ALWAYS_INLINE void call(TTo& result, const TFrom& input) {
+    uint32_t hash = folly::Endian::swap32(
+        folly::hash::SpookyHashV2::Hash32(input.data(), input.size(), 0));
+    static const auto hashLen = sizeof(int32_t);
+    result.resize(hashLen);
+    std::memcpy(result.data(), &hash, hashLen);
+  }
+};
+
+/// spooky_hash_v2_64(varbinary) -> varbinary
+template <typename T>
+struct SpookyHashV2_64Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TTo, typename TFrom>
+  FOLLY_ALWAYS_INLINE void call(TTo& result, const TFrom& input) {
```

Comment:
Please add a comment about the endian-ness from the PR description to here instead. Also note that comments should always end with a full-stop.